### ISSUE TITLE
Package request: Label Studio

### ISSUE THREAD

**Issue by usmcamp0811** [NixOS/nixpkgs#263060]
**Project description**
>"The most flexible data labeling platform to fine-tune LLMs, prepare training data or validate AI models."

**Metadata**

* homepage URL: https://labelstud.io/
* source URL: https://github.com/HumanSignal/label-studio/tree/develop
* license: apache 2.0
* platforms: linux

I tried to get this to work using Dream2Nix but ran into some issues and need help at the very least.  [This](https://gitlab.com/usmcamp0811/campground-packages/-/blob/main/packages/label-studio/default.nix?ref_type=heads) is my current failed* attempt. I can get the Python working* but have issues getting the Node stuff to work correctly.

**Comment by punkeel** [comment]
dupe of #304914

**Comment by eclairevoyant** [comment]
Other way round :)
